contrib/win-installer/build.ps1

```diff
@@ -98,8 +124,10 @@ if ($args.Count -lt 1 -or $args[0].Length -lt 1) {
     Exit 1
 }
 
+InstallWiX
+
 # Pre-set to standard locations in-case build env does not refresh paths
-$Env:Path="$Env:Path;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin;;C:\Program Files\Go\bin"
+$Env:Path="$Env:Path;C:\Program Files (x86)\WiX Toolset *\bin;C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin;;C:\Program Files\Go\bin"
```

Comment:
Drive-by comment: Are you sure this works in powershell?

In *nix the `*` must be on the outside of the quotes otherwise it's taken literally.

Reply:
ack thanks I'll look into this. I've had trouble finding `dotnet` command from the windows image generated in automation_images so I'm working on that for now.